src/java/org/apache/cassandra/config/EncryptionOptions.java

```diff
@@ -330,7 +336,7 @@ public void setEnabled(Boolean enabled) {
      * Return type is Boolean even though it can never be null so that snakeyaml can find it
      * @return if the channel may be encrypted
      */
-    public Boolean isOptional()
+    public Boolean getOptional()
```

Comment:
in java's POJO logic `is` is for `boolean` only, so `Boolean` doesn't get picked up and expects `get` prefix